app/views/layouts/application.html.slim

```diff
@@ -13,6 +13,9 @@ html lang="#{I18n.locale}" class='no-js'
       = content_for?(:title) ? APP_NAME + ' - ' + yield(:title) : APP_NAME
 
     == stylesheet_link_tag 'application', media: 'all'
+    <!--[if IE 8]>
+      <script src='https://cdnjs.cloudflare.com/ajax/libs/es5-shim/4.5.7/es5-shim.min.js'></script>
```

Comment:
We'll need to add this domain to our CSP headers, otherwise it might not be allowed? Well I guess IE8 may not even support CSP so it doesn't matter?